### ISSUE TITLE
UPMC_food101 dataset

### ISSUE THREAD

**Issue by Lixiaomemg** [songrise/MoPE#2]
Hello, are you using the UPMC_food101 dataset? I read that the size of the data set in the article is 90840, but the size of the data set I downloaded is 90688, the size of the training set is 67972, and the size of the test set is 22716. Did you perform other processing on the data set? Can you share the data set you used?

**Comment by songrise** [comment]
Hi, sorry for the delay in response. I did not perform any processing (removing any instances) on the dataset. FYI, I download UPMC Food-101 from this [link](https://github.com/yaoweilee/PMF/issues/1) you can check it out.
